packages/expo-localization/ios/LocalizationModule.swift

```diff
@@ -133,6 +133,20 @@ public class LocalizationModule: Module {
       .map { languageTag -> [String: Any?] in
         let languageLocale = Locale.init(identifier: languageTag)
 
+        if #available(iOS 16, *) {
+          return [
+            "languageTag": languageTag,
+            "languageCode": languageLocale.language.languageCode?.identifier,
```

Comment:
Seems like this shouldn't ever be nullish on any platform.

Reply:
I would assume so as well, but docs seem to state otherwise https://developer.apple.com/documentation/foundation/locale/2293346-languagecode
<img width="528" alt="image" src="https://github.com/expo/expo/assets/5597580/fbfb6cc3-bdd4-4abe-9e9e-9787cdf5ed19">